### ISSUE TITLE
Mobile view - actions on articles do not work with accessibility Features enabled

### ISSUE THREAD

**Issue by joshjama** [gsantner/dandelion#289]
actions on articles does not work with accessibility Features enabled in mobile view. 
In Desktop Mode, if I click on reply or comment, The screenreader says reply or comment - Button. On the mobile view, there is just spelled out the number zero. I can therefore not be sure what to do in mobile view mode as a blind person. 
#### General information

* **App version:** 
* **System:** 
* **Pod:**

**Comment by gsantner** [comment]
does it work when you login at the pod in normal web browser?
